modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/viewmodel/EpisodeListBookmarkViewModel.kt

```diff
@@ -29,9 +25,9 @@ class EpisodeListBookmarkViewModel
             settings.cachedSubscriptionStatus.flow
                 .stateIn(viewModelScope).collect { cachedSubscriptionStatus ->
                     _stateFlow.update { state ->
-                        val userTier = (cachedSubscriptionStatus as? SubscriptionStatus.Paid)?.tier?.toUserTier() ?: UserTier.Free
+                        val isPaidUser = (cachedSubscriptionStatus as? SubscriptionStatus.Paid)?.tier?.isPaid == true
```

Comment:
All such calls will be cleaned up once the `SubscritpionStatus` gets redesigned.